src/scenes/AcceleratorRouter/Deliverables/DeliverableView.tsx

```diff
@@ -64,30 +72,51 @@ const DeliverableViewWrapper = () => {
           setStatus('Not Needed');
           break;
         }
+        case 'download_snapshot': {
+          setShowDownloadModal(true);
+          break;
+        }
       }
     },
     [setStatus]
   );
 
-  const optionItems = useMemo(
+  const speciesOptionItems = useMemo(
     (): DropdownItem[] =>
       activeLocale
         ? [
             {
-              label: strings.formatString(strings.STATUS_WITH_STATUS, strings.NEEDS_TRANSLATION) as string,
-              value: 'needs_translation',
-              disabled: deliverable?.status === 'Needs Translation',
-            },
-            {
-              label: strings.formatString(strings.STATUS_WITH_STATUS, strings.NOT_NEEDED) as string,
-              value: 'not_needed',
-              disabled: deliverable?.status === 'Not Needed',
+              label: strings.DOWNLOAD_SPECIES_SUBMISSION_SNAPSHOT,
+              value: 'download_snapshot',
             },
           ]
         : [],
-    [activeLocale, deliverable?.status]
+    [activeLocale]
   );
 
+  const optionItems = useMemo((): DropdownItem[] => {
+    if (!activeLocale || !deliverable) {
+      return [];
+    }
+
+    const defaultOptionItems = [
+      {
+        label: strings.formatString(strings.STATUS_WITH_STATUS, strings.NEEDS_TRANSLATION) as string,
+        value: 'needs_translation',
+        disabled: deliverable.status === 'Needs Translation',
+      },
+      {
+        label: strings.formatString(strings.STATUS_WITH_STATUS, strings.NOT_NEEDED) as string,
+        value: 'not_needed',
+        disabled: deliverable.status === 'Not Needed',
+      },
+    ];
+
+    return deliverable.type === 'Species' && deliverable.status === 'Approved'
```

Comment:
In questions deliverables previously, there is no option to set deliverable to not-needed or needs-translation. This is changed to give those options, but we can take that away if that is intended.